lib/datachannel.js

```diff
@@ -41,6 +41,11 @@ function RTCDataChannel(internalDC) {
         return this.RTCDataStates[state];
       }
     },
+    'bufferedAmount': {
+      get: function getReadyState() {
```

Comment:
This was pulled in from another patch, so you can revert the implementation for bufferedAmount. If you update your local copy it will be there.